build.gradle.kts

```diff
@@ -19,11 +19,11 @@ dependencies {
 
     implementation("com.squareup.okhttp3:okhttp:4.12.0")
     implementation("org.jetbrains.kotlinx:kotlinx-coroutines-core:1.7.3")
-    implementation("com.connectrpc:connect-kotlin-okhttp:0.5.0")
-    implementation("com.connectrpc:connect-kotlin-google-java-ext:0.5.0")
+    implementation("com.connectrpc:connect-kotlin-okhttp:0.6.0")
+    implementation("com.connectrpc:connect-kotlin-google-java-ext:0.6.0")
     implementation("com.google.protobuf:protobuf-kotlin:3.25.3")
-    implementation("build.buf.gen:impaction-ai_ingestion_connectrpc_kotlin:0.5.0.1.20240304081353.ab10776f8380")
-    implementation("build.buf.gen:impaction-ai_ingestion_protocolbuffers_kotlin:25.3.0.1.20240304081353.ab10776f8380")
+    implementation("build.buf.gen:impaction-ai_ingestion_connectrpc_kotlin:0.7.1.1.20240908234409.342910308823")
```

Comment:
https://buf.build/connectrpc/kotlin?version=v0.7.1

connectrpc/kotlin plugin v0.7.1 recommends to set dependency version to `0.7.1` for connect-kotlin, connect-kotlin-okhttp, connect-kotlin-google-java-ext. Can you double-check the dependency compatibility?